.github/workflows/operator-ci.yml

```diff
@@ -36,4 +36,35 @@ jobs:
 
       - name: Build the Keycloak Operator
         run: |
-          mvn clean package -nsu -B -e -pl operator -Doperator -Dquarkus.container-image.build=true -Dquarkus.kubernetes.deployment-target=minikube
+          mvn clean package -nsu -B -e -pl operator -Doperator -Dquarkus.container-image.build=true -Dquarkus.kubernetes.deployment-target=minikube -DskipTests
+  test:
+    name: test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+      - name: Setup Minikube-Kubernetes
+        uses: manusa/actions-setup-minikube@v2.4.3
+        with:
+          minikube version: v1.22.0
+          kubernetes version: v1.21.1
+          github token: ${{ secrets.GITHUB_TOKEN }}
+          start args: ' --force'
+      - name: Setup Java 11
+        uses: actions/setup-java@v2.3.0
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+          java-package: jdk
+      - name: Build and deploy operator in minikube
+        run: |
+          eval $(minikube -p minikube docker-env)        
+          mvn clean package -Doperator \
+              -Dquarkus.container-image.build=true \
+              -Dquarkus.kubernetes.deploy=true \
+              -Dquarkus.kubernetes.namespace=default \
+              -Dquarkus.kubernetes-client.namespace=default \
+              -Dquarkus.kubernetes.deployment-target=minikube \
+              -DskipTests
+            mvn test -Dtest=OperatorE2ERemoteIT
```

Comment:
I think that we should run `OperatorE2ELocalRunTest` as well.

Reply:
which value would provide ? the end to end is testing exactly the same as the local one, in the "real" version

Reply:
I would say that is a sanity check that the local test keep working as expected for development purposes.

Reply:
`-Dtest=OperatorE2ELocalRunTest,OperatorE2ERemoteIT` should make it

Reply:
doing that you are not testing isolation as one is reusing the other's stuff.
I think the local one is ok to be tested locally, and the final one is the remote one that tests everything closer to the real approach.
To do both tests we would clean the cluster between them.
We can do it, but I would say not important for now, if we think provide value ( I doubt it ) we can add it in the future.

Reply:
I'm okish not running the local (we can get @vmuzikar feedback on this), but your explanation opens one new question to me,
how is this setup going to work when we add more tests other than `OperatorE2ERemoteIT`? Are they supposed to run in the same namespace or different?

> To do both tests we would clean the cluster between them.

It was my understanding that this should be automatically done by the testing framework and/or our usage of it.